components/menu.tsx

```diff
@@ -213,12 +220,59 @@ const SFXMenu = ({state, onSetSFXConfetti: handleSetSFXConfetti, onClose: handle
 	);
 };
 
+const SoundMenu = ({
+	state,
+	onSetSoundEnabled: handleSetSoundEnabled,
+	onClose: handleOnClose,
+}: SoundMenuProperties): React.ReactElement => {
+	return (
+		<div className="fixed flex items-center justify-center inset-0 w-full h-full bg-neutral-100 dark:bg-neutral-900 bg-opacity-80 backdrop-blur-md z-50">
+			<div className="mx-auto w-full max-w-xl">
+				<h2 className="text-neutral-900 dark:text-neutral-100 uppercase text-4xl font-bold">
+					Sound Effects
+				</h2>
+				<div className="mt-6 flex flex-col">
+					<p className="text-neutral-900 dark:text-neutral-100 uppercase text-xs">
+						Enabled
+					</p>
+					<div className="mt-4 flex flex-wrap items-center gap-4">
+						<MenuButton
+							label="Conf"
+							value="ON"
+							theme="green"
+							enabled={state.enableSound ?? true}
+							onClick={handleSetSoundEnabled(true)}
+						/>
+						<MenuButton
+							label="Conf"
+							value="OFF"
+							theme="green"
+							enabled={!(state.enableSound ?? true)}
+							onClick={handleSetSoundEnabled(false)}
+						/>
+					</div>
+				</div>
+				<div className="mt-8 flex flex-col">
+					<button
+						className="w-full px-4 py-2 text-neutral-900 dark:text-neutral-200 bg-neutral-300 dark:bg-neutral-800 hover:bg-neutral-200 dark:hover:bg-neutral-700 border-2 border-neutral-400 dark:border-neutral-700 rounded-md"
+						type="button"
+						onClick={handleOnClose}
+					>
+						Close
+					</button>
+				</div>
+			</div>
+		</div>
+	);
+};
+
```

Comment:
The purpose of the existing SFX (special effects) menu is to house all of these auxiliary features. Please add the sound toggle functionality to it (under the confetti options) and remove this dedicated menu along with the additional sound button from the menu.